addons/mail/static/tests/core/record.test.js

```diff
@@ -1048,3 +1048,47 @@ test("insert with id relation keeps existing field values", async () => {
     expect(member2.channel.eq(channel1)).toBe(true);
     expect(member2.is_internal).toBe(true);
 });
+
+test("accessing to inverse of a computed field should trigger recomputation", async () => {
+    (class Thread extends Record {
+        static id = "id";
+        id;
+        messageToCompute = Record.one("Message", {
+            compute() {
+                expect.step("compute messageToCompute");
+                return this.messages[0];
+            },
+            inverse: "threadOfComputedMessage",
+        });
+        messages = Record.many("Message");
+    }).register(localRegistry);
+
+    (class Message extends Record {
+        static id = "id";
+        id;
+        threadOfComputedMessage = Record.one("Thread", { inverse: "messageToCompute" });
+    }).register(localRegistry);
+
+    const store = await start();
+    const thread = store.Thread.insert({ id: 1 });
+
+    let renderCount = 0;
+    function render() {
+        renderCount++;
+        expect.step(`render ${renderCount}`);
+        for (const message of thread.messages) {
+            message.threadOfComputedMessage;
+        }
+    }
```

Comment:
IMO I'm not fond of assertion of count from notifying reactive. This is implementation details and also makes it hard to debug the test because it's not intuitive why the counter doesn't match (and what may be the cause of issue).

I'd suggest to return list of `toRaw(message.threadOfComputeMessage)._raw.id`, so it's easier to see actual value asserted. The `toRaw()._raw` is there to prevent reactive observe of `.id` that could compromise the purpose of this test.